src/main/java/spring/intro/Main.java

```diff
@@ -0,0 +1,17 @@
+package spring.intro;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import spring.intro.config.AppConfig;
+import spring.intro.model.User;
+import spring.intro.service.UserService;
+
+public class Main {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
```

Comment:
What is the length of this line?

Reply:
101 chars. But clean package says that build was successful...
I split it in two lines